Tasks/PowerShellOnTargetMachines/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 47
+    "Patch": 51
```

Comment:
Why from version 47 to 51 ?
Because we have done hotfix for version 1 from chaitanya [branch](https://github.com/Microsoft/vsts-tasks/tree/users/chshrikh/TaskFixForDTLServiceDeprecation_M123)  not through m123. In chaitanya branch last version for PS on target machine is 50 and window file copy is 42.